File: src/test/java/be/cegeka/orderit/service/eventstore/api/ResultChangedEventTest.java
package be.cegeka.orderit.service.eventstore.api;

import be.cegeka.orderit.service.eventstore.api.AbstractInternalAnalysisEvent;
import be.cegeka.orderit.service.eventstore.api.AnalysisNumber;
import be.cegeka.orderit.service.eventstore.api.InternalAnalysisAggregateId;
import be.cegeka.orderit.service.eventstore.api.ResultChangedEvent;

public class ResultChangedEventTest extends BaseInternalAnalysisEventTest {

    @Override
    protected AbstractInternalAnalysisEvent item() {
        final InternalAnalysisAggregateId aggregateIdentifier = new InternalAnalysisAggregateId("safdghjk");
        return new ResultChangedEvent(aggregateIdentifier, AnalysisNumber.from(45), "23%", "usertje");
    }
}

File: src/test/kotlin/be/cegeka/orderit/service/eventstore/api/ResultChangedEventTest.kt
package be.cegeka.orderit.service.eventstore.api

import be.cegeka.orderit.service.eventstore.api.AnalysisNumber.Companion.from

class ResultChangedEventTest : BaseInternalAnalysisEventTest() {

    override fun item(): AbstractInternalAnalysisEvent {
        val aggregateIdentifier = InternalAnalysisAggregateId("safdghjk")
        return ResultChangedEvent(aggregateIdentifier, from(45), "23%", "usertje")
    }
}